pkg/swarm/swarm.go

```diff
@@ -10,15 +10,23 @@ import (
 	"encoding/hex"
 	"encoding/json"
 	"fmt"
+
+	"golang.org/x/crypto/sha3"
 )
 
 const (
-	SectionSize       = 32
-	Branches          = 128
-	ChunkSize         = SectionSize * Branches
-	HashSize          = 32
-	MaxPO       uint8 = 15
-	MaxBins           = MaxPO + 1
+	SectionSize             = 32
+	Branches                = 128
+	ChunkSize               = SectionSize * Branches
+	HashSize                = 32
+	MaxPO             uint8 = 15
+	MaxBins                 = MaxPO + 1
+	SpanSize                = 8
+	ChunkWithSpanSize       = ChunkSize + SpanSize
+)
+
+var (
+	NewHasher = sha3.NewLegacyKeccak256
```

Comment:
is there any reason why we call it _new_ hasher in the variable? (and not just _hasher_)

Reply:
well the method it aliases also has `New`... in the name

Reply:
i don't think it is a good practice to have words like `new` in the code, since their meaning is questionable, relative, and does not age well in this case.

if it is done in this case to mimic or reference the `sha3` library, that's ok i guess (ftr: i still don't like it :upside_down_face:)

Reply:
thanks @nolash for reaching out and pointing out this was a function and not the actual hasher.

disregard my previous comment, fully support using `new` if we are talking about a constructor :+1: